src/Paket.Core/Versioning/PlatformMatching.fs

```diff
@@ -27,26 +27,26 @@ let inline split (path : string) =
 
 // TODO: This function does now quite a lot, there probably should be several functions.
 let private extractPlatformsPriv = memoize (fun path ->
-    if System.String.IsNullOrEmpty path then Some ParsedPlatformPath.Empty
```

Comment:
move the nullcheck out of memoize, because memoize uses a dictionary and those cannot handle `null`.
Maybe we should extend memoize to support `null`?